<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DspPreferencesWidget</class>
 <widget class="QWidget" name="DspPreferencesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>DSP Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="dspAddBtn">
            <property name="text">
             <string>Add</string>
            </property>
            <property name="icon">
             <iconset theme="list-add">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dspRmBtn">
            <property name="text">
             <string>Delete</string>
            </property>
            <property name="icon">
             <iconset theme="list-remove">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dspConfBtn">
            <property name="text">
             <string>Configure</string>
            </property>
            <property name="icon">
             <iconset theme="settings-configure">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QListView" name="dspListView"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="dspUpBtn">
              <property name="text">
               <string>Up</string>
              </property>
              <property name="icon">
               <iconset theme="arrow-up">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="dspDownBtn">
              <property name="text">
               <string>Down</string>
              </property>
              <property name="icon">
               <iconset theme="arrow-down">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>DSP chain preset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="presetsComboBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="presetSaveBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
          <property name="icon">
           <iconset theme="document-save">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="presetLoadBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
          <property name="icon">
           <iconset theme="document-open">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="presetDelBtn">
          <property name="text">
           <string>Delete</string>
          </property>
          <property name="icon">
           <iconset theme="edit-delete">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
